contracts/LifMarketMaker.sol

```diff
@@ -63,21 +68,28 @@ contract LifMarketMaker is Ownable {
     totalPeriods = _totalPeriods;
     foundationAddr = _foundationAddr;
     initialWei = msg.value;
-    initialBuyPrice = initialWei.div(lifToken.totalSupply());
+    initialBuyPrice = initialWei
+      .mul(PRICE_FACTOR)
+      .div(lifToken.totalSupply().div(PRICE_FACTOR));
```

Comment:
I think we should remove the `div(PRICE_FACTOR)`, just keep the `mul` in previous line